src/fprime_extras/core/util.py

```diff
@@ -29,3 +30,16 @@ def __exit__(self, et, ev, tb):
             self.logger.removeHandler(self.handler)
         if self.handler and self.close:
             self.handler.close()
+
+
+def find_fprime(cwd: Path) -> Path:
```

Comment:
Thanks for including the type hints here, I need to update the rest of the code to have these as well.